.env_example

```diff
@@ -0,0 +1,16 @@
+; Oauth parameters.
+OAUTH_CONSUMER_TOKEN =
+OAUTH_SECRET_TOKEN   =
+OAUTH_ENDPOINT       = 'https://meta.wikimedia.org/w/index.php?title=Special:OAuth'
+OAUTH_REDIR          = 'https://meta.wikimedia.org/wiki/Special:OAuth/authenticate?'
+OAUTH_CALLBACK       = 'oob'
```

Comment:
Interesting. Yeah for us it looks like we need a full URL to `/copypatrol/oauth/callback`. On my local I have the app running at `localhost/copypatrol` so I use `http://localhost/copypatrol/oauth/callback` and this works. We might suggest something similar here